web/src/ClearLogs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import styled from "styled-components"
+import { useLogStore } from "./LogStore"
+import {
+  AnimDuration,
+  Color,
+  FontSize,
+  mixinResetButtonStyle,
+} from "./style-helpers"
+
+const ClearLogsButton = styled.button`
+  ${mixinResetButtonStyle}
+  margin-left: auto;
+  font-size: ${FontSize.small};
+  color: ${Color.white};
+
+  & {
+    transition: color ${AnimDuration.default} ease;
```

Comment:
you can move this with the other properties... "&" doesn't really make sense as a selector, since it's just a self-reference